_data/projects/AutoSort.yml

```
@@ -0,0 +1,10 @@
+name: AutoSort
+desc: Sort by a string or by default attributeA brief description of the project
```

Looks like a copy-paste typo here

```suggestion
desc: Sort by a string or by default attribute
```